.ddev/config.yaml

```diff
@@ -4,7 +4,7 @@ php_version: "8.3"
 webserver_type: apache-fpm
 database:
     type: mariadb
-    version: "10.3"
+    version: "10.11"
```

Comment:
Is this change related to the issue? I think it's unrelated. Can you please revert it if not and create another PR where you'll describe this change, the reason for it and how to test it?